pkg/pgmodel/ingestor/duplicates.go

```diff
@@ -0,0 +1,39 @@
+package ingestor
+
+import (
+	"sync"
+	"time"
+
+	"go.uber.org/atomic"
+
+	"github.com/timescale/promscale/pkg/log"
+	"github.com/timescale/promscale/pkg/pgmodel/metrics"
+)
+
+const reportDuplicatesInterval = time.Minute
+
+var (
+	launchReporterOnce    sync.Once
+	duplicateMetricsTotal atomic.Uint64
+)
+
+func registerDuplicates(duplicateSamples int64) {
+	metrics.DuplicateSamples.Add(float64(duplicateSamples))
+	metrics.DuplicateWrites.Inc()
+}
+
+func reportDuplicates(duplicateMetrics uint64) {
+	duplicateMetricsTotal.Store(duplicateMetrics)
```

Comment:
shouldn't this be an Add?
```suggestion
	duplicateMetricsTotal.Add(duplicateMetrics)
```